week-2/G-conditionals-2/exercise-2.js

```diff
@@ -8,6 +8,14 @@
 */
 
 function studentPassed(grade) {
+  if (grade<=49){
+  return"failed";
+  }
+  if (grade>=50){
+return"passed";
+  }if (grade==100){
```

Comment:
Will the condition `grade == 100` ever happen? Trace through the function assuming `grade` is 100 and work out what the function will do.